server/index.js

```diff
@@ -43,16 +42,14 @@ app.use(
     credentials: true,
   })
 );
-app.enable("trust proxy");
 app.use(
   session({
-    secret: sessionSecret,
+    secret: "sessionSecret",
```

Comment:
While this may work, it actually undermines the purpose of our sessions `secret` value. The goal is to use a string that is _inaccessible_ to anyone trying to break in. If we turn this into a string, it can easily be found here in our source code. But if we keep it in an environment variable (`process.env`), it will never be exposed anywhere.

What the actual string is isn't particularly important, it is used to **compute the hash** (that is then stored in a cookie in the browser), and simply needs to be kept secret to be secure.

This question has some good info in the answers - https://stackoverflow.com/questions/5343131/what-is-the-sessions-secret-option

Reply:
Considering that ^, I don't think the changes to the `index.js` file should be merged in, but we'll want to keep diagnosing as to what is the specific problem